### ISSUE TITLE
Suppress "collection_count"

### ISSUE THREAD

**Issue by jvine** [projectblacklight/arclight#836]
Part of #820

In some contexts, the search results display `<div class='al-collection-count'>` above the results. 

- It's redundant with the results count that's already present on the page
- It looks sort of like a heading, but semantically it's just text that's not really connected to anything
- It doesn't work correctly with the "group by collection" functionality (when a result set of collections is grouped by collection - the count goes to 0)

I propose suppressing / removing this section rather than trying to make it work functionally/semantically - it doesn't really add value to the page.

**Comment by mejackreed** [comment]
Fixed in #875
